pages/signup.js

```diff
@@ -78,33 +84,116 @@ export default function Signup() {
+            <TextField size='small' type='password' placeholder='Password' sx={{width:'100%', backgroundColor: '#F5F5F5', mb:1}} onChange={(e) => { handleSignUpFormInput(e.target.value, 'password'), handlePasswordLength() }}></TextField>
+            <Typography sx={{ fontWight: 'light', fontSize: 10, fontStyle: 'italic', color: passwordLengthColor }}  >*minimum password length of 6 characters</Typography>
+            <Button sx={{ my: 2 }} type='submit' className={classes.loginButton} disabled={filledFormFlag}>Create Account</Button>
+            <Modal open={googleOpen} onClose={handleGoogleClose}>
+              <FormControl sx={{
+                position: 'absolute',
+                alignItems: 'center',
+                top: '50%',
+                left: '50%',
+                transform: 'translate(-50%, -50%)',
+                width: 400,
+                bgcolor: 'background.paper',
+                borderRadius: 5,
+                boxShadow: 24,
+                p: 4,
+              }}>
+                <Typography sx={{fontSize: 28}}>Google Signup</Typography>
+                <InputLabel id="accountDropDown"></InputLabel>
+                <Select
+                  labelId='accountDropDown'
+                  id="accountDropDown"
+                  value={accountType}
+                  sx={{ height: 30, margin: 2, width: 336}}
+                  onChange={(e) => { handleSignUpFormInput(e.target.value, 'account_type'), setAccountType(e.target.value) }}
+                >
+                  <MenuItem value={'Mentor'}>Mentor</MenuItem>
+                  <MenuItem value={'Mentee'}>Mentee</MenuItem>
+                </Select>
+                <Stack direction="row" spacing={1}>
+                  <TextField size='small' variant="filled" placeholder='First name' sx={{
+                    my: .3,
+                    borderRadius: 5,
+                    width: '50%',
+                  }} required onChange={(e) => handleSignUpFormInput(e.target.value, 'firstname')}></TextField>
+                  <TextField size='small' variant="filled" placeholder='Last name' sx={{
+                    my: .3,
+                    borderRadius: 5,
+                    width: '50%',
+                  }} required onChange={(e) => handleSignUpFormInput(e.target.value, 'lastname')}></TextField>
+                </Stack>
+                <TextField size='small' variant="filled" placeholder='Location' sx={{ my: .3, width: 337}}  required onChange={(e) => handleSignUpFormInput(e.target.value, 'location')}></TextField>
+                <Button sx={{ my: 2,  }} onClick={(e) => signInWithGoogle(signupInfo)} className={classes.moduleGoogleButton} startIcon={<GoogleIcon />}>Sign in with Google</Button>
+              </FormControl>
+            </Modal>
+            <Modal open={facebookOpen} onClose={handleFacebookClose}>
```

Comment:
Do we need 2 different modal forms? Is there a way to use 1 modal form and conditionally render if its FB or google based on what button we press?